zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala

```diff
@@ -359,6 +359,9 @@ private[consumer] final class Runloop private (
                          pollResult.records
                        )
       updatedPendingCommits <- ZIO.filter(state.pendingCommits)(_.isPending)
+      _ <- ZIO.whenZIO(ZIO.exists(updatedStreams) { stream =>
+             ZIO.ifZIO(stream.maxPollIntervalExceeded)(stream.halted.as(true), ZIO.succeed(false))
```

Comment:
To make things more explicit I'd prefer that we use named arguments here:
```suggestion
             ZIO.ifZIO(stream.maxPollIntervalExceeded)(onTrue = stream.halted.as(true), onFalse = ZIO.succeed(false))
```

Reply:
I have used the 'more elegant way' in https://github.com/zio/zio-kafka/pull/1009/commits/1cfd33bf2ee86ffd30ccda7e8077c64069db7f9e.